parser/prism/Translator.cc

```diff
@@ -3955,15 +3969,15 @@ ast::ExpressionPtr Translator::desugarArray(core::LocOffsets location, absl::Spa
 //   * PM_KEYWORD_HASH_NODE (keyword arguments to a method call)
 //
 // @param elements The Prism key/value pairs to be translated
-parser::NodeVec Translator::translateKeyValuePairs(pm_node_list_t elements) {
+parser::NodeVec Translator::translateKeyValuePairs(pm_node_list_t elements, bool forKwargsHash) {
```

Comment:
It doesn't look like `forKwargsHash` is used, is that intentional?

Reply:
Pulled out. I'm expecting the CI to pass, so you can auto-merge